### PR TITLE
Fixed travis.yml, busdevice

### PR DESCRIPTION
**Pull request by kattni**

### CODE REVIEW

**Comment by kattni** [comment]
Need to add dependency PCA9685.

**Comment by kattni** [comment]
Also needs INA219. On hold until those are setup.

**Comment by kattni** [comment]
Dependencies added, failing on Sphinx now. `Adafruit-Blinka` should be handling the failure, I'm not entirely sure why it's not. Could be that it's using a shared I2C setup. It's possible we could skip this one, but I would hook a FeatherWing up to a RPi, so I assume other people will as well, and these helper libs are great. They'll eventually be expanded to include more wings as well. So it might be worth figuring out why it's not working.

File: docs/conf.py [hunk 1]
```diff
@@ -18,13 +18,16 @@
...
+autodoc_mock_imports = ["adafruit_featherwing", "shared"]
 
 intersphinx_mapping = {
     'python': ('https://docs.python.org/3.4', None),
     'BusDevice': ('https://circuitpython.readthedocs.io/projects/bus_device/en/latest/', None),
     'Register': ('https://circuitpython.readthedocs.io/projects/register/en/latest/', None),
     'CircuitPython': ('https://circuitpython.readthedocs.io/en/latest/', None),
-    'motor': ('https://circuitpython.readthedocs.io/projects/motor/en/latest/', None)}
+    'motor': ('https://circuitpython.readthedocs.io/projects/motor/en/latest/', None),
+    'seesae': ('https://circuitpython.readthedocs.io/projects/seesaw/en/latest/', None),
```
Comment (sommersoft) [inline docs/conf.py]: typo...'seesaw`: (don't think this is related to Travis failure)

Comment (kattni) [reply docs/conf.py]: I noticed this and had already fixed it, hadn't pushed yet.

**Comment by kattni** [comment]
I'm not convinced this won't break the docs. It looks weird building locally, but it always looks weird building locally so I can't tell whether it's a local issue or whether it'll carry to RTD. Giving `api.rst` a title means the modules don't show up on the left column. You'd have to click the title to get to the page with the modules. Locally, the modules don't show up on the `api.rst` page, instead it says `Used by autodoc_mock_imports` three times. There is a chance that this will also cause that failure to occur on RTD.

I guess we merge it and see what happens.

**Pull request merged**